### PR TITLE
Revised documentation files

### LINKED ISSUE #2091: Update docs and add Azure DevOps integration
### Description
*We should add an example in the integrations docs for using KICS with Azure DevOps pipelines*

### PR DESCRIPTION
**Pull request by nunoocx**
Closes #2091

**Proposed Changes**

- Complete revision of documentation
- Revised Integrations section
- Added integration section for Azure Pipelines

I submit this contribution under Apache-2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,15 @@ Support of other solutions and additional cloud providers are on the [roadmap](d
 
 Setting up and using KICS is super-easy.
 
--   First, see how to [install and get KICS running](docs/getting-started.md).
--   Next, check how you can easily [integrate it into your CI](docs/integrations.md) for any project.
--   Eventually, [explore the output results format](docs/results.md) and quickly fix the issues detected.
+Setting up and using KICS is super-easy.
```
Comment (ruigomescx) [inline README.md:32]: This sentence is repeated

**Review by ruigomescx — CHANGES_REQUESTED** [review]

File: docs/integrations_azurepipelines.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          ${TARGET_DIR}/kics --no-progress -q ${TARGET_DIR}/assets/queries -p ${PWD} -o ${PWD}/kics-results.json
+
+          TOTAL_SEVERITY_COUNTER=`grep '"totalCounter"':' ' kics-results.json | awk {'print $2'}`
+          export SEVERITY_COUNTER_HIGH=`grep '"HIGH"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+          SEVERITY_COUNTER_MEDIUM=`grep '"INFO"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+          SEVERITY_COUNTER_LOW=`grep '"LOW"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+          SEVERITY_COUNTER_INFO=`grep '"MEDIUM"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+          echo "TOTAL SEVERITY COUNTER $TOTAL_SEVERITY_COUNTER"
+
+          if [
```
Comment (rogeriopeixotocx) [inline docs/integrations_azurepipelines.md:114]: missing if condition

File: docs/integrations_azurepipelines.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+stages:
+- stage: Kics
+  displayName: Kics
+  # variables: # several syntaxes, see specific section
```
Comment (rogeriopeixotocx) [inline docs/integrations_azurepipelines.md:82]: this comment is not needed

File: docs/integrations_azurepipelines.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+export SEVERITY_COUNTER_HIGH=`grep '"HIGH"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+SEVERITY_COUNTER_MEDIUM=`grep '"INFO"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+SEVERITY_COUNTER_LOW=`grep '"LOW"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+SEVERITY_COUNTER_INFO=`grep '"MEDIUM"':' ' kics-results.json | awk {'print $2'} | sed 's/.$//'`
+echo "TOTAL SEVERITY COUNTER $TOTAL_SEVERITY_COUNTER"
+```
+
+3.6- Optionally, define a breaking point for the CI
+```shell
+if [
```
Comment (rogeriopeixotocx) [inline docs/integrations_azurepipelines.md:63]: missing if condition

File: docs/integrations_azurepipelines.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+          PACKAGE_NAME=kics_${LATEST_VERSION}_${OS}_x64.tar.gz
+          TARGET_DIR=/home/vsts/kics
+
+          mkdir -p ${TARGET_DIR}
+          wget -q -c https://github.com/Checkmarx/kics/releases/download/${LATEST_TAG}/${PACKAGE_NAME} -O - | tar -xz -C ${TARGET_DIR}
+
+          echo '--- START SCANNING ---'
+          ${TARGET_DIR}/kics --no-progress -q ${TARGET_DIR}/assets/queries -p ${PWD} -o ${PWD}/kics-results.json
+
+          TOTAL_SEVERITY_COUNTER=`grep '"totalCounter"':' ' kics-results.json | awk {'print $2'}`
```
Comment (rogeriopeixotocx) [inline docs/integrations_azurepipelines.md:107]: `"total_counter"`

File: docs/integrations_gitlabci.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+    paths:
+      - kics-results.json
+```
+
+5- Declare the consumption of results
+```yaml
+kics-results:
+  stage: kics-result
+  before_script:
+    - export TOTAL_SEVERITY_COUNTER=`grep '"totalCounter"':' ' kics-results.json | awk {'print $2'}`
```
Comment (rogeriopeixotocx) [inline docs/integrations_gitlabci.md:49]: `"total_counter"`

File: docs/integrations_azurepipelines.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+3.4- Start the scan
+```shell
+ ${TARGET_DIR}/kics --no-progress -q ${TARGET_DIR}/assets/queries -p ${PWD} -o ${PWD}/kics-results.json
+```
+
+3.5- Consume the results
+```shell
+TOTAL_SEVERITY_COUNTER=`grep '"totalCounter"':' ' kics-results.json | awk {'print $2'}`
```
Comment (rogeriopeixotocx) [inline docs/integrations_azurepipelines.md:53]: `"total_counter"`

File: docs/integrations_azurepipelines.md (hunk 4 above)
Comment (rogeriopeixotocx) [reply docs/integrations_azurepipelines.md:63]: `if [ "$SEVERITY_COUNTER_HIGH" -ge "1" ]; then`

File: docs/integrations_azurepipelines.md (hunk 2 above)
Comment (rogeriopeixotocx) [reply docs/integrations_azurepipelines.md:114]: `if [ "$SEVERITY_COUNTER_HIGH" -ge "1" ]; then`

File: docs/integrations_gitlabci.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+    - kics scan -q /usr/bin/assets/queries -p ${PWD} -o ${PWD}/kics-results.json
+  artifacts:
+    name: kics-results.json
+    paths:
+      - kics-results.json
+
+kics-results:
+  stage: kics-result
+  before_script:
+    - export TOTAL_SEVERITY_COUNTER=`grep '"totalCounter"':' ' kics-results.json | awk {'print $2'}`
```
Comment (rogeriopeixotocx) [inline docs/integrations_gitlabci.md:98]: `"total_counter"`

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Comment by ruigomescx** [comment]
Codacy is reporting the following issues:
https://app.codacy.com/gh/Checkmarx/kics/pullRequest?prid=6982531

**Review by IgorMarkov — APPROVED** [review]

**Review by IgorMarkov — APPROVED** [review]

**Review by IgorMarkov — APPROVED** [review]

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**